models/issues/milestone.go

```diff
@@ -326,6 +380,15 @@ func DeleteMilestoneByRepoID(repoID, id int64) error {
 	if _, err = db.Exec(ctx, "UPDATE `issue` SET milestone_id = 0 WHERE milestone_id = ?", m.ID); err != nil {
 		return err
 	}
+
+	if err = m.LoadLabelsWithContext(ctx); err != nil {
```

Comment:
So I think merge the 2 functions `LoadLabels` and `LoadLabelsWithContext` as one is a good choice.